EventListener/InvalidationSubscriber.php

```diff
@@ -220,11 +226,60 @@ private function invalidateRoutes(array $routes, Request $request)
                 }
             }
 
+            if (null === $route->getParams()) {
+                $typeInvalidation = "BAN";
+            }
+
+            $typeInvalidation = $route->type ? $route->type : $typeInvalidation;
+
+            if ($typeInvalidation === "BAN") {
+                $this->cacheManager->invalidateRegex($this->generateRegexPath($route->getName(), $params));
+
+                continue;
+            }
+
             $this->cacheManager->invalidateRoute($route->getName(), $params);
         }
     }
 
     /**
+     * Set all route params as widlcard by default
+     *
+     * @param string $routeName
+     *
+     * @return array
+     */
+    protected function preFillParams($routeName)
+    {
+        $params = array();
+        $route = $this->router->getRouteCollection()->get($routeName->getName());
```

Comment:
please don't do this. Using ``->getRouteCollection()`` at runtime kills your performance, because it triggers the loading of routes each time